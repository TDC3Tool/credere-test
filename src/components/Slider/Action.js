import React, { useContext, useCallback } from 'react'
import Button from '../Button'
import { SliderButton } from './styles'
import SliderContext from './context'

const SliderAction = () => {
  const { activatedSlider, setActivatedSlider, maxSlider } = useContext(SliderContext)

  const handleSetNewSlider = useCallback(() => {
    if (activatedSlider < maxSlider) {
      setActivatedSlider(activatedSlider + 1)
    } else {
      setActivatedSlider(0)
    }
  }, [setActivatedSlider, maxSlider, activatedSlider])

  return (
    <SliderButton className="absolute" data-max={maxSlider}>
      <Button onClick={handleSetNewSlider}>
        Pr√≥ximo
      </Button>
    </SliderButton>
  )
}

export default SliderAction
