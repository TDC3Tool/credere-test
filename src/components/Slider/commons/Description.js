import React from 'react'
import PropTypes from 'prop-types'
import Typography from '../../Typography'
import { DescriptionWrapper } from './styles'

const Description = ({ text, color }) => (text ? (
  <DescriptionWrapper className="flex-column-center">
    <Typography tag="paragraph" color={color}>
      {text}
    </Typography>
  </DescriptionWrapper>
) : null)

Description.propTypes = {
  text: PropTypes.string,
  color: PropTypes.string,
}

Description.defaultProps = {
  color: '#FFFFFF',
  text: null,
}

export default Description
