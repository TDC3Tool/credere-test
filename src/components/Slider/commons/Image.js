import React from 'react'
import classnames from 'classnames'
import PropTypes from 'prop-types'
import { ImageShadow, ImageContent } from './styles'

const Image = ({ src, children, className }) => (
  <div className={classnames(className, 'relative')} style={{ backgroundImage: `url(${src})` }}>
    <ImageShadow className="full-content absolute" />
    {children && (
      <ImageContent className="relative">
        {children}
      </ImageContent>
    )}
  </div>
)

Image.propTypes = {
  src: PropTypes.string.isRequired,
  children: PropTypes.node,
  className: PropTypes.string,
}

Image.defaultProps = {
  children: null,
  className: null,
}

export default Image
