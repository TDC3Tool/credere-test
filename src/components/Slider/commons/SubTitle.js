import React from 'react'
import PropTypes from 'prop-types'
import Typography from '../../Typography'

const SubTitle = ({ text, color }) => (text ? (
  <Typography tag="sub-heading" color={color}>
    {text}
  </Typography>
) : null)

SubTitle.propTypes = {
  color: PropTypes.string,
  text: PropTypes.string,
}

SubTitle.defaultProps = {
  color: '#FFFFFF',
  text: null,
}

export default SubTitle
