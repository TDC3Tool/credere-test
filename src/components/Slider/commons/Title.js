import React from 'react'
import PropTypes from 'prop-types'
import Typography from '../../Typography'

const Title = ({ text, color }) => (
  <Typography tag="heading" color={color} emphasys>
    {text}
  </Typography>
)

Title.propTypes = {
  text: PropTypes.string.isRequired,
  color: PropTypes.string,
}

Title.defaultProps = {
  color: '#FFFFFF',
}

export default Title
