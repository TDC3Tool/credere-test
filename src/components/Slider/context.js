import React, { createContext, useState, useEffect } from 'react'
import PropTypes from 'prop-types'

const SliderContext = createContext({
  activatedSlider: 0,
  maxSlider: 0,
  setMaxSlider: () => {},
  setActivatedSlider: () => {},
})

const SliderContextProvider = ({ children, count }) => {
  const [activatedSlider, setActivatedSlider] = useState(0)
  const [maxSlider, setMaxSlider] = useState(0)

  useEffect(() => {
    setMaxSlider(count)
  }, [setMaxSlider, count])

  return (
    <SliderContext.Provider value={{
      activatedSlider,
      maxSlider,
      setMaxSlider,
      setActivatedSlider,
    }}
    >
      {children}
    </SliderContext.Provider>
  )
}

SliderContextProvider.propTypes = {
  children: PropTypes.node.isRequired,
  count: PropTypes.number,
}

SliderContextProvider.defaultProps = {
  count: 1,
}

export {
  SliderContextProvider,
}

export default SliderContext
