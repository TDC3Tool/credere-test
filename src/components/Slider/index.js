import React from 'react'
import PropTypes from 'prop-types'
import Item from './Item'

const Slider = ({
  duration, effect, autoPlay, children,
}) => {
  console.log(duration, effect, autoPlay)

  return (
    <div>
      {children}
    </div>
  )
}

Slider.propTypes = {
  children: PropTypes.node.isRequired,
  duration: PropTypes.number,
  effect: PropTypes.oneOf(['fade-in', 'slide-in-left']),
  autoPlay: PropTypes.bool,
}

Slider.defaultProps = {
  duration: 500,
  effect: 'slide-in-left',
  autoPlay: false,
}

Slider.Item = Item

export default Slider
