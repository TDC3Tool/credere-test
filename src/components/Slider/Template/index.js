import React from 'react'
import PropTypes from 'prop-types'
import Image from '../commons/Image'
import Title from '../commons/Title'
import SubTitle from '../commons/SubTitle'
import Description from '../commons/Description'
import {
  DefaultTemplate, ThirdTemplate, FirstTemplate, FirstTemplateHeading,
} from './styles'

const First = ({
  // eslint-disable-next-line react/prop-types
  image, title, subTitle, description,
}) => (
  <FirstTemplate className="full-content grid">
    <Image className="flex-algin-end full-content" src={image} />
    <div>
      <FirstTemplateHeading>
        <Title text={title} color="#333333" />
        <SubTitle text={subTitle} color="#999999" />
      </FirstTemplateHeading>
      <Description text={description} color="#333333" />
    </div>
  </FirstTemplate>
)

const Second = ({
  // eslint-disable-next-line react/prop-types
  image, title, subTitle,
}) => (
  <div className="full-content">
    <Image className="flex-algin-end full-content" src={image}>
      <Title text={title} />
      <SubTitle text={subTitle} />
    </Image>
  </div>
)


const Third = ({
  // eslint-disable-next-line react/prop-types
  image, title, subTitle,
}) => (
  <ThirdTemplate className="full-content">
    <Image className="flex-column-center full-content" src={image}>
      <Title text={title} />
      <SubTitle text={subTitle} />
    </Image>
  </ThirdTemplate>
)

const Default = ({
  // eslint-disable-next-line react/prop-types
  image, title, subTitle, description,
}) => (
  <DefaultTemplate className="grid full-content">
    <Image className="flex-column-center" src={image}>
      <Title text={title} />
      <SubTitle text={subTitle} />
    </Image>
    <Description text={description} color="#333333" />
  </DefaultTemplate>
)

const Template = ({ ...props }) => {
  switch (props.template) {
    case 'default':
      return <Default {...props} />

    case 'template_3':
      return <Third {...props} />

    case 'template_2':
      return <Second {...props} />

    case 'template_1':
      return <First {...props} />

    default:
      return <Default {...props} />
  }
}

Template.propTypes = {
  template: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  subTitle: PropTypes.string,
  description: PropTypes.string,
}

Template.defaultProps = {
  subTitle: null,
  description: null,
}

export default Template
