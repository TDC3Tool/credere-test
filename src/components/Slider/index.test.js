import React from 'react'
import renderer from 'react-test-renderer'
import Slider from '.'
import ButtonAction from './Action'
import Description from './commons/Description'
import Image from './commons/Image'
import SubTitle from './commons/SubTitle'
import Title from './commons/Title'
import Template from './Template'

const slide = {
  image: 'https://img.ibxk.com.br/2019/07/10/10091423089001.jpg',
  title: 'Lorem ipsum dolor',
  subTitle: 'Sed non dui sed purus hendrerit aliquet a nec magna. ',
  description: 'Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis purus eleifend non',
}

describe('Slider Component Snapshots', () => {
  it('should render slider component', () => {
    const tree = renderer.create(
      <>
        <Slider>
          <Slider.Item template="default" index={1} {...slide} />
          <Slider.Item template="template_1" index={2} {...slide} />
          <Slider.Item template="template_2" index={3} {...slide} />
          <Slider.Item template="template_3" index={4} {...slide} />
        </Slider>
      </>,
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('should render slider component without childrens', () => {
    const element = renderer.create(<Slider />).toJSON()
    expect(element).toMatchSnapshot()
  })

  it('should render slider item', () => {
    const element = renderer.create(<Slider.Item template="default" index={1} {...slide} />).toJSON()
    expect(element).toMatchSnapshot()
  })

  it('should render button action', () => {
    const element = renderer.create(<ButtonAction />).toJSON()
    expect(element).toMatchSnapshot()
  })

  it('should render common description', () => {
    const tree = renderer.create(
      <>
        <Description />
        <Description text={slide.description} color="#333" />
      </>,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should render common title', () => {
    const element = renderer.create(<Title text={slide.title} color="#333" />).toJSON()
    expect(element).toMatchSnapshot()
  })

  it('should render common sub title', () => {
    const element = renderer.create(
      <>
        <SubTitle />
        <SubTitle text={slide.title} color="#333" />
      </>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })

  it('should render common image', () => {
    const tree = renderer.create(
      <>
        <Image src={slide.image} />
        <Image src={slide.image}>Children</Image>
      </>,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should render template list options', () => {
    const tree = renderer.create(
      <>
        <Template {...slide} />
        <Template template="default" {...slide} />
        <Template template="template_1" {...slide} />
        <Template template="template_2" {...slide} />
        <Template template="template_3" {...slide} />
      </>,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
