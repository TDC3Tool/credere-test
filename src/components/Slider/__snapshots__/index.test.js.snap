// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider Component Snapshots should render button action 1`] = `
<div
  className="absolute sc-bxivhb dEKikf"
  data-max={0}
>
  <button
    className="transition-all sc-cSHVUG bRWBVW"
    onClick={[Function]}
    type="button"
  >
    <span
      className="sc-gZMcBi giNnHm"
      style={
        Object {
          "color": "#FFFFFF",
        }
      }
    >
      Próximo
    </span>
  </button>
</div>
`;

exports[`Slider Component Snapshots should render common description 1`] = `
<div
  className="flex-column-center sc-gzVnrw dkdosU"
>
  <p
    className="sc-iwsKbI ibDZNz"
    style={
      Object {
        "color": "#333",
      }
    }
  >
    Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
  </p>
</div>
`;

exports[`Slider Component Snapshots should render common image 1`] = `
Array [
  <div
    className="relative"
    style={
      Object {
        "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
      }
    }
  >
    <div
      className="full-content absolute sc-EHOje fZgOQP"
    />
  </div>,
  <div
    className="relative"
    style={
      Object {
        "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
      }
    }
  >
    <div
      className="full-content absolute sc-EHOje fZgOQP"
    />
    <div
      className="relative sc-bZQynM sMRAj"
    >
      Children
    </div>
  </div>,
]
`;

exports[`Slider Component Snapshots should render common sub title 1`] = `
<h3
  className="sc-dnqmqq bJaIPW"
  style={
    Object {
      "color": "#333",
    }
  }
>
  Lorem ipsum dolor
</h3>
`;

exports[`Slider Component Snapshots should render common title 1`] = `
<h2
  className="sc-htoDjs kexmgL"
  style={
    Object {
      "color": "#333",
    }
  }
>
  Lorem ipsum dolor
</h2>
`;

exports[`Slider Component Snapshots should render slider component 1`] = `
<div
  className="relative sc-bdVaJa bJmnKC"
>
  <div
    className="full-content relative sc-bwzfXH uTxCW"
  >
    <div
      className="hidden transition-all absolute sc-htpNat dxOMxF"
      data-index={1}
      data-template="default"
    >
      <div
        className="grid full-content sc-gqjmRU kIqdMS"
      >
        <div
          className="flex-column-center relative"
          style={
            Object {
              "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
            }
          }
        >
          <div
            className="full-content absolute sc-EHOje fZgOQP"
          />
          <div
            className="relative sc-bZQynM sMRAj"
          >
            <h2
              className="sc-htoDjs kexmgL"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Lorem ipsum dolor
            </h2>
            <h3
              className="sc-dnqmqq bJaIPW"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Sed non dui sed purus hendrerit aliquet a nec magna. 
            </h3>
          </div>
        </div>
        <div
          className="flex-column-center sc-gzVnrw dkdosU"
        >
          <p
            className="sc-iwsKbI ibDZNz"
            style={
              Object {
                "color": "#333333",
              }
            }
          >
            Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
          </p>
        </div>
      </div>
    </div>
    <div
      className="hidden transition-all absolute sc-htpNat dxOMxF"
      data-index={2}
      data-template="template_1"
    >
      <div
        className="full-content grid sc-jTzLTM hfhqii"
      >
        <div
          className="flex-algin-end full-content relative"
          style={
            Object {
              "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
            }
          }
        >
          <div
            className="full-content absolute sc-EHOje fZgOQP"
          />
        </div>
        <div>
          <div
            className="sc-fjdhpX dJvaXu"
          >
            <h2
              className="sc-htoDjs kexmgL"
              style={
                Object {
                  "color": "#333333",
                }
              }
            >
              Lorem ipsum dolor
            </h2>
            <h3
              className="sc-dnqmqq bJaIPW"
              style={
                Object {
                  "color": "#999999",
                }
              }
            >
              Sed non dui sed purus hendrerit aliquet a nec magna. 
            </h3>
          </div>
          <div
            className="flex-column-center sc-gzVnrw dkdosU"
          >
            <p
              className="sc-iwsKbI ibDZNz"
              style={
                Object {
                  "color": "#333333",
                }
              }
            >
              Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="hidden transition-all absolute sc-htpNat dxOMxF"
      data-index={3}
      data-template="template_2"
    >
      <div
        className="full-content sc-jzJRlG bhbMmg"
      >
        <div
          className="flex-algin-end full-content relative"
          style={
            Object {
              "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
            }
          }
        >
          <div
            className="full-content absolute sc-EHOje fZgOQP"
          />
          <div
            className="relative sc-bZQynM sMRAj"
          >
            <h2
              className="sc-htoDjs kexmgL"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Lorem ipsum dolor
            </h2>
            <h3
              className="sc-dnqmqq bJaIPW"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Sed non dui sed purus hendrerit aliquet a nec magna. 
            </h3>
          </div>
        </div>
      </div>
    </div>
    <div
      className="hidden transition-all absolute sc-htpNat dxOMxF"
      data-index={4}
      data-template="template_3"
    >
      <div
        className="full-content sc-VigVT gNgpkb"
      >
        <div
          className="flex-column-center full-content relative"
          style={
            Object {
              "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
            }
          }
        >
          <div
            className="full-content absolute sc-EHOje fZgOQP"
          />
          <div
            className="relative sc-bZQynM sMRAj"
          >
            <h2
              className="sc-htoDjs kexmgL"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Lorem ipsum dolor
            </h2>
            <h3
              className="sc-dnqmqq bJaIPW"
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Sed non dui sed purus hendrerit aliquet a nec magna. 
            </h3>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="absolute sc-bxivhb dEKikf"
    data-max={0}
  >
    <button
      className="transition-all sc-cSHVUG bRWBVW"
      onClick={[Function]}
      type="button"
    >
      <span
        className="sc-gZMcBi giNnHm"
        style={
          Object {
            "color": "#FFFFFF",
          }
        }
      >
        Próximo
      </span>
    </button>
  </div>
</div>
`;

exports[`Slider Component Snapshots should render slider component without childrens 1`] = `null`;

exports[`Slider Component Snapshots should render slider item 1`] = `
<div
  className="hidden transition-all absolute sc-htpNat dxOMxF"
  data-index={1}
  data-template="default"
>
  <div
    className="grid full-content sc-gqjmRU kIqdMS"
  >
    <div
      className="flex-column-center relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
      <div
        className="relative sc-bZQynM sMRAj"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
    </div>
    <div
      className="flex-column-center sc-gzVnrw dkdosU"
    >
      <p
        className="sc-iwsKbI ibDZNz"
        style={
          Object {
            "color": "#333333",
          }
        }
      >
        Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
      </p>
    </div>
  </div>
</div>
`;

exports[`Slider Component Snapshots should render template list options 1`] = `
Array [
  <div
    className="grid full-content sc-gqjmRU kIqdMS"
  >
    <div
      className="flex-column-center relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
      <div
        className="relative sc-bZQynM sMRAj"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
    </div>
    <div
      className="flex-column-center sc-gzVnrw dkdosU"
    >
      <p
        className="sc-iwsKbI ibDZNz"
        style={
          Object {
            "color": "#333333",
          }
        }
      >
        Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
      </p>
    </div>
  </div>,
  <div
    className="grid full-content sc-gqjmRU kIqdMS"
  >
    <div
      className="flex-column-center relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
      <div
        className="relative sc-bZQynM sMRAj"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
    </div>
    <div
      className="flex-column-center sc-gzVnrw dkdosU"
    >
      <p
        className="sc-iwsKbI ibDZNz"
        style={
          Object {
            "color": "#333333",
          }
        }
      >
        Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
      </p>
    </div>
  </div>,
  <div
    className="full-content grid sc-jTzLTM hfhqii"
  >
    <div
      className="flex-algin-end full-content relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
    </div>
    <div>
      <div
        className="sc-fjdhpX dJvaXu"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#333333",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#999999",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
      <div
        className="flex-column-center sc-gzVnrw dkdosU"
      >
        <p
          className="sc-iwsKbI ibDZNz"
          style={
            Object {
              "color": "#333333",
            }
          }
        >
          Aliquam facilisis tempus neque, ac accumsan elit bibendum non. Duis molestie libero porta tortor placerat, vitae cursus diam placerat. Fusce feugiat libero erat, suscipit venenatis...
        </p>
      </div>
    </div>
  </div>,
  <div
    className="full-content sc-jzJRlG bhbMmg"
  >
    <div
      className="flex-algin-end full-content relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
      <div
        className="relative sc-bZQynM sMRAj"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
    </div>
  </div>,
  <div
    className="full-content sc-VigVT gNgpkb"
  >
    <div
      className="flex-column-center full-content relative"
      style={
        Object {
          "backgroundImage": "url(https://img.ibxk.com.br/2019/07/10/10091423089001.jpg)",
        }
      }
    >
      <div
        className="full-content absolute sc-EHOje fZgOQP"
      />
      <div
        className="relative sc-bZQynM sMRAj"
      >
        <h2
          className="sc-htoDjs kexmgL"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Lorem ipsum dolor
        </h2>
        <h3
          className="sc-dnqmqq bJaIPW"
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Sed non dui sed purus hendrerit aliquet a nec magna. 
        </h3>
      </div>
    </div>
  </div>,
]
`;
